packages/expo-camera/ios/ExpoCameraUtils.swift

```diff
@@ -0,0 +1,178 @@
+import AVFoundation
+import CoreMotion
+
+struct ExpoCameraUtils {
+  static func device(with mediaType: AVMediaType, preferring position: AVCaptureDevice.Position) -> AVCaptureDevice? {
+    return AVCaptureDevice.default(.builtInWideAngleCamera, for: mediaType, position: position)
+  }
+
+  static func deviceOrientation(
+    for accelerometerData: CMAccelerometerData,
+    default orientation: UIDeviceOrientation
+    ) -> UIDeviceOrientation {
```

Comment:
```suggestion
  ) -> UIDeviceOrientation {
```

But it should be fine to collapse, the signature is not too long 🤔

Reply:
Had a very hard time with swiftlint on this one. My local setup gives different warnings than CI. Might need to talk to you about it.